### PR TITLE
Feat: add `StatusNotifierItemProxy::activate` & `StatusNotifierItemProxy::secondary_activate`

### PR DESCRIPTION
**Pull request by ogios**
these are typically used when left&middle click on the icon.

for more information, i found this:
https://github.com/Rayzeq/tryfol/blob/bd70ee5c2e699c26003cc0c719ca3c312d4fa219/tryfol-bar/src/backend/status_notifier/proxy/item.rs#L78

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -521,6 +532,32 @@ impl Client {
...
+            .build()
+            .await?;
+        Ok(proxy)
+    }
+
+    async fn get_menu_proxy(
+        &self,
+        address: String,
+        menu_path: String,
+        // ) -> Result<DBusMenuProxy<'_>, zbus::Error> {
```
Comment (JakeStanger) [inline src/client.rs:551]: can this please be removed please

**Comment by JakeStanger** [comment]
Thanks, this looks like a solid addition to me!

File: src/client.rs (hunk 1 above)
Comment (ogios) [reply src/client.rs]: removed. i forgot this _(:з」∠)_

**Review by JakeStanger — APPROVED** [review]
lgtm, cheers. I'll stick a new release out to include this soon.

**Pull request merged**